robottelo/ui/locators/base.py

```diff
@@ -245,7 +245,9 @@
         "//a[contains(@data-url, '/compute_resources/test_connection')]"),
     "resource.username": (By.ID, "compute_resource_user"),
     "resource.password": (By.ID, "compute_resource_password"),
-    "resource.datacenter": (By.XPATH, "//select[@id='compute_resource_uuid']"),
+    "resource.datacenter": (By.XPATH, "//div[contains"
+                            "(@id, 's2id_compute_resource_uuid')]"
+                            "/a/span[contains(@class, 'arrow')]"),
```

Comment:
Also, let's update the locator as below for consistency sake and for better readability,

```
"resource.datacenter": (
    By.XPATH, 
    ("//div[contains(@id, 's2id_compute_resource_uuid')]"
     "/a/span[contains(@class, 'arrow')]")),
```